src/projectStatus.ts

```diff
@@ -773,6 +929,52 @@ class TestPreset extends Node {
     }
 }
 
+class PackagePreset extends Node {
+
+    async initialize(): Promise<void> {
```

Comment:
Is there a reason we don't have code that lets us know that there are cpackArgs or cpackEnvironment settings that might override? Similar to the Configure/Build/Test presets?

Reply:
Also, assuming you add this, make sure it reacts to real time settings changes.

Reply:
I did not realize that was missing after the last merge conflicts of bringing over latest functionality.